linux/usage/systemd.md

```diff
@@ -26,12 +26,18 @@ WantedBy=multi-user.target
 ```
 So in this instance, the service would run Python 3 from our working directory `/home/pi/myscript` which contains our python program to run `main.py`. But you are not limited to Python programs: simply change the ExecStart line to be the command to start any program/script that you want running from booting.
 
-Copy this file into `/etc/systemd/system` as root, for example:
+Copy this file into `/var/run/systemd/system` as root, for example:
```

Comment:
The path was fine as it was. You should not be telling users to put configuration files under `/var/run`, since they will disappear on reboot.

(To be 100% correct the path should probably be `/etc/systemd/user` rather than `system`).

Reply:
http://manpages.ubuntu.com/manpages/xenial/en/man5/systemd.unit.5.html /etc/systemd or /run/systemd which is a symlink to /run.

Reply:
> http://manpages.ubuntu.com/manpages/xenial/en/man5/systemd.unit.5.html /etc/systemd or /run/systemd which is a symlink to /run.

That documentation is for Ubuntu 16: on Raspberry Pi OS `/etc/systemd` and `/var/run/systemd` are two different things - they're not symlinked.

Reply:
then I will remove the change